StreamAPI/.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file and other settings should not be a part of the pull request. Add `.idea` to the `.gitignore `file of the project (StreamAPI).